### PR TITLE
Updating the documentation for experimental i18n models

### PR DESCRIPTION
**Pull request by mariepellat**
Added pointers for using the fr, es and de models.

### CODE REVIEW

File: api_reference.md [hunk 1]
```diff
@@ -120,6 +122,27 @@ thoroughly.
...
+interested in testing an experimental model and are willing to change your code once the
+production models are available. These models are experimental and have not been tested as
+thoroughly as their English counterparts.
+
+*   TOXICITY_EXPERIMENTAL
+*   SEVERE_TOXICITY_EXPERIMENTAL
+*   PROFANITY_EXPERIMENTAL
+*   IDENTITY_ATTACK_EXPERIMENTAL
+*   INSULT_EXPERIMENTAL
+*   THREAT_EXPERIMENTAL
```
Comment (jetpack) [inline api_reference.md]: Could you bold the model names with `**` to fit the same style as the other model names?

File: api_reference.md [hunk 2]
```diff
@@ -120,6 +122,27 @@ thoroughly.
...
+
+Please refer to the rest of the documentation bellow to appropriately set the
+language fields in your request.
```
Comment (jetpack) [inline api_reference.md]: Could you also update the docs for the `languages` field (below)? It currently says "Currently, all models only support English. Explicitly specifying a language other than "en" will fail. If the language autodetection determines the comment is in a non-English language, the request also fails."

File: api_reference.md [hunk 3]
```diff
@@ -120,6 +122,27 @@ thoroughly.
...
+
```
Comment (jetpack) [inline api_reference.md:146]: I think it's worth explicitly mentioning the difficulties of calling the API if the user doesn't know the language. Namely, if the user has data in both Spanish and English (for example), there's no way to make a single API call that automatically uses the right model depending on the language.

We can just say something like "Currently, if your comments are in both English and another experimentally-supported language (Spanish, French, German), there's no simple way to use the API that works for all comments unfortunately. You can either (1) change how you call the API depending on the comment language, or (2) wait until these models become "Alpha", when they'll be handled automatically."

File: api_reference.md (hunk 2 above)
Comment (mariepellat) [reply api_reference.md]: thanks for catching this.

File: api_reference.md (hunk 3 above)
Comment (mariepellat) [reply api_reference.md:146]: Great suggestion.

**Comment by mariepellat** [comment]
@jetpack final comments ?

File: api_reference.md [hunk 4]
```diff
@@ -245,7 +249,7 @@ Field | Description
...
+`languages`              | *(optional)* A list of [ISO 631-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) two-letter language codes specifying the language(s) that `comment` is in (for example, "en", "es", "fr", "de", "zh", etc). If unspecified, the API will autodetect the comment language. If language detection fails, the API returns an error. *Note:* Currently, all alpha models only support English and some experimental toxicity models support French, Spanish and German. Explicitly specifying a language other than "en" will fail. If the language autodetection determines the comment is in a non-English language, the request also fails.
```
Comment (jetpack) [inline api_reference.md]: This part isn't accurate: `Explicitly specifying a language other than "en" will fail. If the language autodetection determines the comment is in a non-English language, the request also fails.`

Maybe just remove those 2 sentences now?

**Review by jetpack — APPROVED** [review]
Just one remaining inaccuracy. Feel free to push once addressed.

**Comment by iislucas** [comment]
LGTM modulo Jetpack's last comment.

File: api_reference.md (hunk 4 above)
Comment (mariepellat) [reply api_reference.md]: addressed this.

Comment (mariepellat) [reply api_reference.md]: Done.

**Pull request merged**